README.md

```diff
@@ -221,9 +246,9 @@ OpenStack | N | N | Y | N | N
 oVirt | N | N | Y | N | N
 VMWare | N | N | Y | N | N
```

Comment:
It would be nice to add alibaba cloud to support matrix here. I think we missed it.

Reply:
Sure; I didn't try to change any content with this PR, except in the one place where I knew (by trying it) that the new `oc adm release extract` flags were available.

Reply:
makes sense. we can keep that for later.